### ISSUE TITLE
Can I set workDirectory when using gobuild tasks?

### ISSUE THREAD

**Issue by JackTiger** [gogradle/gogradle#284]
Can I set workdir when using gobuild task?like as below:

workingDir "xxxxx"

**Comment by blindpirate** [comment]
No, it's impossible currently: https://github.com/gogradle/gogradle/blob/1fb5682b474c025d89677676d91032ac5f38bad5/src/main/java/com/github/blindpirate/gogradle/build/DefaultBuildManager.java#L241

The working directory of forked processes are hardcoded to be `project.getProjectDir()`.

**Comment by JackTiger** [comment]
I find a problem when I run gobuild in windows,  After I Build I get the message as below:

`The filename or extension is too long`

So I want to set gobuild workDirectory to avoid this problem.

**Comment by blindpirate** [comment]
I'm not sure setting working directory would help. Can you share some examples to reproduce `The filename or extension is too long`? Maybe it's caused by something else, or we can fix it in other ways.
